setuptools/_importlib.py

```diff
@@ -1,12 +1,7 @@
+import importlib.resources as resources  # noqa: F401
 import sys
 
 if sys.version_info < (3, 10):
     import importlib_metadata as metadata  # pragma: no cover
 else:
     import importlib.metadata as metadata  # noqa: F401
-
-
-if sys.version_info < (3, 9):
-    import importlib_resources as resources  # pragma: no cover
-else:
-    import importlib.resources as resources  # noqa: F401
```

Comment:
I think that if we don't do this change, we might be able to keep the PRs thematically separated... let me see if we can do that.